moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -1184,40 +1184,40 @@ class MoveGroup::MoveGroupImpl
 }
 }
 
-moveit::planning_interface::MoveGroup::MoveGroup(const std::string &group_name, const boost::shared_ptr<tf::Transformer> &tf, const ros::WallDuration &wait_for_servers)
+moveit::planning_interface::MoveGroupInterface::MoveGroupInterface(const std::string &group_name, const boost::shared_ptr<tf::Transformer> &tf, const ros::WallDuration &wait_for_servers)
```

Comment:
> you see that the object file provides the symbol ``foo::func``

is that bad? is your code example showing what you *should not* do? because that is what i was advocating we should do.

From the syntax examples on [cppreference](http://en.cppreference.com/w/cpp/language/namespace) I am advocating #1 and you are advocating we stick with #4, correct/